### ISSUE TITLE
Bot not ascending after major goal achieved

### ISSUE THREAD

**Issue by DarkPulse** [prinzstani/CookieBot#48]
For some reason, the bot isn't ascending once it achieves its major goal, on either the normal or Beta versions of the bot.

I saw this, and read up on what it does once it prepares to ascend - namely, pop all Wrinklers, sell all buildings, buy the Chocolate Cookie, and then ascend.

So I went into the Stats menu and (manually) selected to pop all the wrinklers, but before I could do the rest of the steps, the bot immediately completed them.

This leads me to believe that something related to the wrinklers is making the bot "stuck" in terms of ascending.

**Comment by prinzstani** [comment]
Thanks for describing this problem. Unfortunately, it is very difficult to see what happens without more detail. Please send an export save if this problem reoccurs.
The main reason for the bot not to ascend is that it waits for plants to be ripe. This can be seems in the status description (when you hover over the cookie clicker version line).

**Comment by DarkPulse** [comment]
I don't believe I had the plant stuff as an option at the time (I'm a long-lapsed player who saw the massive updates, so I found my old save and imported it). I do have it now, though (and the other stuff now that I actually gained three sugar lumps), and aside from that manual reset, it seems to have managed to auto-reset when it hits its major goal just fine since then.

So now I'm not exactly sure what's caused it. I thought it might have to do with it putting the game in Christmas season by the time it should ascend, but it's been doing that just fine too, so I'm stumped.

For what it's worth, it didn't indicate it was waiting on anything else, and no subgoals were listed - I remember specifically all it said was "preparing to ascend."
